src/e2e-test/java/io/cdap/cdap/ui/utils/Constants.java

```diff
@@ -32,6 +32,9 @@ private static double getRandomArbitrary(int min, int max) {
   public static final String CONFIGURATION_URL = "http://localhost:11011/cdap/administration/configuration";
   public static final String SYSTEM_PROFILES_CREATE_URL = "http://localhost:11011/cdap/ns/system/profiles/create";
   public static final String DEFAULT_PROFILES_CREATE_URL = "http://localhost:11011/cdap/ns/default/profiles/create";
+  public static final String CONTROL_CENTER_URL = "http://localhost:11011/cdap/ns/default/control";
```

Comment:
You don't need to add a constant for every URL. you can create a base studio URL like `http://localhost:11011/cdap/ns/default/` and use `BASE_STUDIO_URL + 'profiles/create'`

Reply:
Addressed